spec/rack/test/utils_spec.rb

```diff
@@ -66,7 +66,7 @@
         :input => StringIO.new(data)
       }
       env = Rack::MockRequest.env_for("/", options)
-      params = Rack::Utils::Multipart.parse_multipart(env)
+      params = Rack::Multipart.parse_multipart(env)
```

Comment:
@perlun yes it exists in Rack 1.x too.
This modification is because `rack` project have just deleted the code to access from `Rack::Utils` to `Multipart` from rack 2.x.

https://github.com/rack/rack/commit/b2d73960e9ea6b8b15321ef190f13a290d1aedf0

```
$ git show b2d73960e9ea6b8b15321ef190f13a290d1aedf0 lib/rack/utils.rb
...
-    Multipart = Rack::Multipart
-
...
```